pom.xml

```diff
@@ -778,6 +778,12 @@
             <groupId>com.google.gwt.gwtmockito</groupId>
             <artifactId>gwtmockito</artifactId>
             <version>${gwtmockito.version}</version>
+            <exclusions>
```

Comment:
Not sure if we still need `gwtmockito` as dependency. All which made use of `GwtMockitoTestRunner` are gone. Do I missed something?